keyboards/handwired/dactyl_manuform/dmote/62key/keymaps/default/keymap.c

```diff
@@ -0,0 +1,152 @@
+#include "62key.h"
+#include "rgblight.h"
+#include <keymap_colemak.h>
+#include <sendstring_colemak.h>
+
+extern keymap_config_t keymap_config;
+
+#define MODS_SHIFT_MASK (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT))
```

Comment:
## Lines 8-10:

QMK defines macros for all of these in the core code. If you use the built-in names, you don't need to define these here at all.

https://github.com/qmk/qmk_firmware/blob/48b01446ba6b74d7e0793f972873a10fceef2f62/tmk_core/common/keycode.h#L49-L63